src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
```

Comment:
```suggestion
    protected String color;
```
add one-arg constructor and setter